examples/ez-food/order-processor/Ballerina.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+org = "example"
+name = "orderprocessor"
```

Comment:
```suggestion
name = "order_processor"
```

Reply:
This is also a valid naming convention which can be used with ballerina packages.